pkg/stanza/operator/input/journald/config_all.go

```diff
@@ -27,16 +27,18 @@ func NewConfigWithID(operatorID string) *Config {
 type Config struct {
 	helper.InputConfig `mapstructure:",squash"`
 
-	Directory   *string       `mapstructure:"directory,omitempty"`
-	Files       []string      `mapstructure:"files,omitempty"`
-	StartAt     string        `mapstructure:"start_at,omitempty"`
-	Units       []string      `mapstructure:"units,omitempty"`
-	Priority    string        `mapstructure:"priority,omitempty"`
-	Matches     []MatchConfig `mapstructure:"matches,omitempty"`
-	Identifiers []string      `mapstructure:"identifiers,omitempty"`
-	Grep        string        `mapstructure:"grep,omitempty"`
-	Dmesg       bool          `mapstructure:"dmesg,omitempty"`
-	All         bool          `mapstructure:"all,omitempty"`
+	Directory           *string       `mapstructure:"directory,omitempty"`
+	Files               []string      `mapstructure:"files,omitempty"`
+	StartAt             string        `mapstructure:"start_at,omitempty"`
+	Units               []string      `mapstructure:"units,omitempty"`
+	Priority            string        `mapstructure:"priority,omitempty"`
+	Matches             []MatchConfig `mapstructure:"matches,omitempty"`
+	Identifiers         []string      `mapstructure:"identifiers,omitempty"`
+	Grep                string        `mapstructure:"grep,omitempty"`
+	Dmesg               bool          `mapstructure:"dmesg,omitempty"`
+	All                 bool          `mapstructure:"all,omitempty"`
+	ConvertMessageBytes bool          `mapstructure:"convert_message_bytes,omitempty"`
```

Comment:
As to the format of the `MESSAGE` field, [systemd's docs](https://github.com/systemd/systemd/blob/main/docs/JOURNAL_EXPORT_FORMATS.md#journal-json-format) yield the following:

> A field that contains non-printable or non-UTF8 is serialized as a number array instead. This is necessary to handle binary data in a safe way without losing data, since JSON cannot embed binary data natively. Each byte of the binary field will be mapped to its numeric value in the range 0…255.

Consulting golang's `encoding/json` `Unmarshal` method [docs](https://pkg.go.dev/encoding/json#Unmarshal), what is supposed to be `[]byte` is presented as `[]interface{}` containing `float64`s. I'm not familiar enough with Go to know a better typecast procedure.

> To unmarshal JSON into an interface value, Unmarshal stores one of these in the interface value:
> [...]
> - float64, for JSON numbers
> [...]
> - []interface{}, for JSON arrays

I think giving the user the choice of what to do in that particular case is a good idea, as backends vary and some might want the message record as it was written by `journald` to be passed on.